sirepo/api.py

```diff
@@ -0,0 +1,15 @@
+class APIRequest:
```

Comment:
Add our standard file header.

Also document the class. It's holds the request context for all API calls. On reading this, I think we should rename this module to sirepo.request and the class would be called Base. I comment on how to automate these changes.

Reply:
3ccfbc6e956420f7da6fc23176cdebb7ae57cfc9